CHANGELOG.md

```diff
@@ -1,7 +1,15 @@
 
 # Alfred API - Changelog
 
-## 2.7.1 (Unreleased)
+## 2.7.2 (Unreleased)
```

Comment:
The 2.7.2 part needs to be in the post-release commit (https://xenitsupport.jira.com/wiki/spaces/XEN/pages/758480899/XEP-11+Release+Procedure), it must not end up in `release-2x`.